pkg/backup/backupschedule/backup_schedule_manager.go

```diff
@@ -249,6 +275,44 @@ func buildBackup(bs *v1alpha1.BackupSchedule, timestamp time.Time) *v1alpha1.Bac
 	return backup
 }
 
+func buildLogBackup(bs *v1alpha1.BackupSchedule, timestamp time.Time) *v1alpha1.Backup {
+	ns := bs.GetNamespace()
+	bsName := bs.GetName()
+
+	logBackupSpec := *bs.Spec.LogBackupTemplate.DeepCopy()
```

Comment:
Is it possible to encounter a nil error here? when `LogBackupTemplate` = nil

Reply:
ok, but `performLogBackupIfNeeded` will check `LogBackupTemplate`, if it's nil, will not build log backup. add a check here is fine too.